### PR TITLE
Fix possible bug wellborepath layer

### PR DESCRIPTION
**Pull request by nicholasdalhaug**
Issue: #362

### CODE REVIEW

**Review by Brynjulf — CHANGES_REQUESTED** [review]
Your solution to part one works well.

Part two has been mischaracterised by us as a bug when it is actually a feature to clamp any values outside the curve to the curve. The real issue is to document this, which will be a separate task (#370)

So if you remove your change related to part two and clean up the imports I can approve this pull request

File: .storybook/src/features/highlight.stories.ts [hunk 1]
```diff
@@ -106,8 +106,8 @@ export const HighlightWellborepathWithController = () => {
     // external event that calls the rescale event the highlighting should change
     const slider = createSlider((event: any) => onUpdate(event, { rescaleEvent: controller.currentStateAsEvent, layer: highlightLayer }), {
       width,
-      min: -1000,
-      max: controller.referenceSystem.length + 1000,
+      min: 0,
+      max: controller.referenceSystem.length,
```
[the remark this answers is not in the document]
Comment (nicholasdalhaug) [reply .storybook/src/features/highlight.stories.ts:110]: Are you sure it is not clamped the wrong way in that case? I see why one might want the slider to not move the point all the way to the end or start. I do not see why you would want the slider itself to not go all the way to its start or end. Anyways, I will fix it.

File: .storybook/src/features/highlight.stories.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import { Selection } from 'd3-selection';
 
-import { WellborepathLayer, HTMLLayer, GridLayer } from '../../../src/layers';
-import { OnRescaleEvent, OnMountEvent } from '../../../src/interfaces';
+import { WellborepathLayer, HTMLLayer } from '../../../src/layers';
+import { OnRescaleEvent, OnMountEvent } from '../../../src';
 import { IntersectionReferenceSystem, Controller } from '../../../src/control';
-import { ZoomPanHandler } from '../../../src/control/ZoomPanHandler';
+import { ZoomPanHandler } from '../../../src/control';
```
[the remark this answers is not in the document]
Comment (nicholasdalhaug) [reply .storybook/src/features/highlight.stories.ts:6]: Yes, I did not realise that. Thanks!

**Comment by nicholasdalhaug** [comment]
Could you do me the favor of labelling this PR with "hacktoberfest-accepted" if you think it is a valid contribution to the project?

File: .storybook/src/features/highlight.stories.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Brynjulf) [reply .storybook/src/features/highlight.stories.ts]: The slider is meant to illustrate a one-dimensional view and not the curve, in our actual use case we needed extensions to be able to handle panning past the end of the curve. We also used a mouse move event to get two-dimensional coordinates to highlight the point along the curve in other parts of our application. This also means that we needed to handle values that were outside the curve gracefully.  

That being said, there are things that can be done to improve this example but is outside the scope of this pull request at least

**Review by Brynjulf — APPROVED** [review]
Looks good

**Comment by Brynjulf** [comment]
@nicholasdalhaug done. Thank you for your contribution, we appreciate it 😊

**Pull request merged**